src/pages/Sign/sections/SignSubscription.tsx

```diff
@@ -301,6 +308,12 @@ export const SubscriptionComponent: React.SFC<SubscriptionComponentProps> = ({
                       currentOffer.insurance.type,
                     )
                   }}
+                  runStudentkortet={() =>
```

Comment:
Maybe we should figure out if they are from studentkortet first? based on utm-tags? so that we don't load scripts for all users.

Reply:
Oh, lol my bad 😆Now checking _hvcode